BurnOutSharp/ProtectionType/CDLock.cs

```diff
@@ -44,7 +64,12 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
         {
             var matchers = new List<PathMatchSet>
             {
-                new PathMatchSet(new PathMatch(".AFP", useEndsWith: true), "CD-Lock"),
+                // TODO: Determine if there's any consistency in the naming of the additional AFP files.
+
+                // Found in every confirmed sample of CD-Lock, generally (but not always) appears to include markers relating to the additional AFP files present (Redump entries 24287 and 31615). 
+                new PathMatchSet(new PathMatch("CONFIG.AFP", useEndsWith: true), "CD-Lock"),
```

Comment:
I hope the answer is "no", but is there any possibility that there could be a false positive with something like `appconfig.afp` from another program? Or do the odds of that seem super low?

Reply:
If there's a viable way to scan for only "CONFIG.AFP", I'd be happy to change it to that. But the original check was for *only* the AFP extension, which seemingly hasn't caused any issues for false positives as of yet. So I think the chances for a false positive for this is pretty low.

Reply:
Good enough for me.